demo/api/src/config/config.ts

```diff
@@ -69,6 +69,9 @@ export function createConfig(processEnv: NodeJS.ProcessEnv) {
         ecgRtrList: {
             apiKey: envVars.ECG_RTR_LIST_API_KEY,
         },
+        encryption: {
+            encryptionKey: envVars.ENCRYPTION_KEY,
```

Comment:
```suggestion
            key: envVars.ENCRYPTION_KEY,
```

I would either nest it with encryption.key or just write encryptionKey